### ISSUE TITLE
Project 4: incorrect categories of outcome 

### ISSUE THREAD

**Issue by kaseyzapatka** [NalinaSofia/Computational-Social-Science-Projects#3]
So, the main issue you are having that I can gather from our meeting during office hours that you are `HSD010` outcome, which has 5 or 6 categories and instead of converting the labels to numbers like this: 

>  excellent     = 1
>  very good   = 2
>  good           = 3
>  fair              = 4
>  poor            = 5

You're doing something like this that binarizes each value and creates an array for each possible value: 

>  excellent     = 1,0,0,0,0
>  very good   = 0,1,0,0,0
>  good           = 0,0,1,0,0
>  fair              = 0,0,0,1,0
>  poor            = 0,0,0,0,1

This is a problems because you are working with a data frame where each case should only have 1 value but for this variable, each row has 5 values. For example a case that is poor has `0,0,0,0,1` instead of `5`. 

I also think you are making this more complicated by using the multiple category instead of the binary, which is simply `good` or `bad`, `1` or `0`. 

So, I'd use something like the code below (it starts in your workflow around code chunk 18) to reformat the outcome variable to be 1, 0 instead of labels (good, bad). You'll have to determine whether you want good as 1 or bad as 1--it will depend on what you want to be predicting. 

```nhanes_scaled['HSD010_binary_numeric'] = nhanes_scaled['HSD010_binary'].replace(
["good", "poor"], [0,1]) ```


Once you get that make sure to use that as your outcome. and you'll want to fit the neural net with num_class = 2 . I would use code that looks like this to fit the neural net. Just make sure `num_classes` actually equals 2. 

```
num_classes = y_train.shape[1]

model = Sequential()

# hidden layer 
model.add(Dense(num_classes, input_dim= X_train.shape[1], kernel_initializer= 'normal', activation= 'relu'))

# output layer 
model.add(Dense(num_classes, kernel_initializer= 'normal', activation= 'softmax'))

## Add any additional layers you wish here
model.compile(loss= 'categorical_crossentropy', optimizer= 'adam', metrics=['accuracy'])

model.fit(X_train, y_train, validation_data=(X_test, y_test), epochs=10, batch_size=200, verbose=2)

```
Let me know if you still have questions. You'll want to cut some of the extra code that you have to ensure that you are using the right outcome. 

Good luck

**Comment by NalinaSofia** [comment]
Thanks Kasey
